Lib/test/test_logging.py

```diff
@@ -4086,6 +4086,13 @@ def test_invalid_names(self):
         self.assertRaises(TypeError, logging.getLogger, any)
         self.assertRaises(TypeError, logging.getLogger, b'foo')
 
+    def test_pickling(self):
+        for name in ('', 'root', 'foo', 'foo.bar', 'baz.bar'):
```

Comment:
Is `self.logger` pickleable? If not, it may make sense to add an explicit test for this.